client/packages/android/app/src/main/assets/public/cordova.js

```diff
@@ -0,0 +1,1689 @@
+// Platform: Capacitor
```

Comment:
this is committed to reduce setup steps

Reply:
It could work, it just makes it more complicated, what if postinstall hook fails ?